### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by jedbrown**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](/help/leaving-feedback-in-github)”.
</details>

### CODE REVIEW

**Comment by Yunhan1** [comment]
Good work here, it is pretty like the regression we have in this course. How do you know that it does not converge to sin(x) when getting the high degree?

**Comment by ghost** [comment]
Pretty good work looking at sklearn's `PolynomialFeatures` class. 

Since the Taylor series expansion for your function `f(x)=x*sin(x)` is: `x^2 - x^4/3! + x^6/5! - x^8/7! + ...` I'm thinking that since the terms will just go to zero/machine epsilon super quickly that sklearn will ignore them.  I'd like to know if this is the case or not.

**Comment by pokemm** [comment]
It looks pretty cool overall. 

Would using cosspace instead of linspace sampling points change the overall behavior of sklearn?

**Comment by techLord-pm** [comment]
I enjoyed the overview of Sklearn and the implementation of matrices and regression. I am curious if there is a range for which the program works while balancing accuracy and unpredictability.

**Comment by ghost** [comment]
Good usage of sklearn's large library. It's interesting to note that the a higher degree leads to a higher accuracy in the overfitting problem.

**Comment by lbakken** [comment]
I think machine learning is an excellent field to demonstrate the real-world applications of what we learned this semester! Thank you for including a demonstrative graphic and detailed summary; they augmented the project as a whole.

**Comment by harshdesh7** [comment]
Cool project. How did the matrices generated by the PolynomialFeatures class compare to other vandermonde matrices?
